app/models/account.rb

```diff
@@ -48,7 +48,8 @@ def self.tenants(tenant_list)
             unless: :cname_is_blank?
 
   validates :tenant, presence: true, uniqueness: true
-  validates :cname, presence: true, uniqueness: true, exclusion: { in: [default_cname('')] }
+  validates :cname, presence: true, uniqueness: true,
+                    exclusion: { in: Account.excluded_cnames.map { |c| default_cname(c) } }
```

Comment:
Tests are failing because `excluded_cnames` isn't defined at this point.  Does it need to be a lambda or proc?  Or maybe changing `excluded_cnames` into a constant or just defining it before this `validates`?